packages/Button/test/Button.spec.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+
```

Comment:
✂️

Reply:
Oh, actually I need this for these parts:
https://github.com/acl-services/paprika/blob/5d4d70aea056924b7e4bd9ffbfa80527d584bdc6/packages/Button/test/Button.spec.js#L74
https://github.com/acl-services/paprika/blob/5d4d70aea056924b7e4bd9ffbfa80527d584bdc6/packages/Button/test/Button.spec.js#L88